guides/v2.1/config-guide/multi-site/ms_over.md

```diff
@@ -16,57 +12,43 @@ functional_areas:
   - Setup
 ---
 
-## Introduction to multiple Magento stores and websites {#ms-over}
+A single instance of the {{site.data.var.ee}} software enables you to start multiple websites or store views that use different attributes and content, such as:
 
-One instance of the Magento software can enable you to start multiple websites or store views that use different:
+-  Default languages
+-  Domain names
+-  Categories
+-  Products
+-  Currencies
 
-*   Default languages
-*   {% glossarytooltip 41aee03b-a5d5-49c2-8839-894090ef4e86 %}Domain{% endglossarytooltip %} names
-*   Categories
-*   Products
-*   Currency
+This flexible solution enables one Magento codebase and {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} to administer and display different stores. You configure the websites, stores, and store views in the Magento {% glossarytooltip 29ddb393-ca22-4df9-a8d4-0024d75739b1 %}Admin{% endglossarytooltip %}. Use certain variables in virtual hosts to start the Magento application using these websites or store views. 
```

Comment:
I don't think it's necessary to use a glossary tooltip for a term twice in a paragraph. One time is sufficient.